test/commands/list/activities_spec.rb

```diff
@@ -52,8 +52,39 @@
           stdout_only "2014-12-31: Celebrated the new year in Paris with Marie Curie. @partying"
         end
       end
-    end
 
+      describe "when implicit location is set" do
```

Comment:
This test is very thorough, nice! Would you mind also adding our implicit location case to [this test](https://github.com/JacobEvelyn/friends/blob/master/test/commands/clean_spec.rb#L72) to ensure that if we've never seen the location before it gets added?

Reply:
Hey @JacobEvelyn, could you expand on what you mean by moving our implcit location here? Implicit locations don't seem to have anything to do with cleaning the `friends.md` file - any `moved to _LOCATION_` lines and lines with no location in their description would be read and treated exactly the same as the lines already included in this test, and any implcit locations set on any of the activities would not have an effect on what is stored in the `friends.md` file.

Reply:
Ah, good question. So any time info is added we [call `clean` internally to re-write the file](https://github.com/JacobEvelyn/friends/blob/master/bin/friends#L97). And when a new location is mentioned in an added activity, it's [automatically added to the location list](https://github.com/JacobEvelyn/friends/issues/182). So I'd just like a test to confirm that this behavior still happens when the new location is first mentioned in a `moved to _New Place_` activity.

Does that make sense?

Reply:
I will push back slightly: from my understanding, a `moved to _LOCATION_` line will be treated by the `clean` method in exactly the same way as any other line containing _LOCATION_ (the regex cares about the _LOCATION_ part, and not the `moved to` part), hence perhaps there is no need to add a 'moved to _LOCATION_ line'. But, it is a small change and the test remains read-able so I've added this to the test :)

Reply:
Yup makes sense! I haven't thought about regression much in the projects I've worked on, but I guess there is balancing act - you don't want to write test absolutely everything in your when you add a feature, but you want a safety net when it is probable a feature could touch other parts of the code. I need more exposure to walking that fine line :)